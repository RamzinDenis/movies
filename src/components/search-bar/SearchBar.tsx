import React from "react";
import styles from "./search-bar.module.sass";
import { SearchOutlined } from "@ant-design/icons";

const SearchBar: React.FC = () => {
	return (
		<form className={styles.form}>
			<input className={styles.input} placeholder="Search Movie"></input>
			<button className={styles.button}>
				<SearchOutlined className={styles.icon} />
			</button>
		</form>
	);
};

export default SearchBar;
