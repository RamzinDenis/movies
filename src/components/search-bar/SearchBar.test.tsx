import React from "react";
import { shallow } from "enzyme";
import SearchBar from "./";
import toJson from "enzyme-to-json";

describe("<SearchBar />", () => {
	it("should render SearchBar properly", () => {
		expect(toJson(shallow(<SearchBar />))).toMatchSnapshot();
	});
});
