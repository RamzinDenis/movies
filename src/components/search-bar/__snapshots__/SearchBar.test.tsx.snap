// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> should render SearchBar properly 1`] = `
<form
  className="form"
>
  <input
    className="input"
    placeholder="Search Movie"
  />
  <button
    className="button"
  >
    <ForwardRef(SearchOutlined)
      className="icon"
    />
  </button>
</form>
`;
